src/frontend/package.json

```diff
@@ -41,9 +41,11 @@
     "@mdi/font": "^7.2.96",
     "@ng-bootstrap/ng-bootstrap": "^14.2.0",
     "@ngneat/transloco": "^4.2.7",
+    "@types/mocha": "^10.0.1",
```

Comment:
Such entries should be added to `devDependencies` the next time because they are only needed during development and not by the app itself. I'll correct that in a different PR.